doc/Support/Device-Sensors.md

```diff
@@ -15,7 +15,7 @@ High and Low values. List of these sensors can be found here [Link](../Developin
 
 ### Wireless Sensors
 
-Some Wireless have  High and Low values of these sensors can be edited in Web UI by going to the device settings -> Health. There you can set your own custom High and Low values. List of these sensors can be found here [Link](../Developing/os/Wireless-Sensors.md)
+Some Wireless have  High and Low values of these sensors can be edited in Web UI by going to the device settings -> Health and Wireless Settings There you can set your own custom High and Low values. List of these sensors can be found here [Link](../Developing/os/Wireless-Sensors.md)
```

Comment:
Should this not just be Settings -> Wireless Sensors?

Reply:
Yes sir you are correct, I will change it.